### PR TITLE
New York Times Article Page Clone

### PR DESCRIPTION
**Pull request by ElijahTobs**
**NYT Clone**

- add navbar
- add header
- add sidebar
- add footer
- use float, flex and grid

### CODE REVIEW

**Review by christianotieno — COMMENTED** [review]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @ElijahTobs  & @ebeagusamuel  :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
Let's improve on the already awesome implementation of the navbar:
From this:
header_clone
A bit of tweaking to make it appear more like this:
header

### NOTES: I am conducting this review with a (1920 x 1080) screen display resolution. So bear with me if your resolution might differ with mines. You are free to reach out to me here or on slack for further discussion on the same.
The big main centred image and text appears not to be properly centred. 
- This is how it is appearing on my screen:
image_clone

- This is how the original appears on my screen:
image

The same also applies to the editor's pick section:
Yours:
editor_clone

Original:
editor

- Next in line, we have the 'More in Space and Astronomy' section:
Yours:
more_space_clone

Original:
more_space

- Lastly, we have the footer section:
- Yours:
footer_clone

- Original:
footer

I think the cause of this would be how we implemented the responsiveness in this project. The page appears to not also be responsive in medium and small screens.  

## [REQUIRED] - VALIDATE -The markup.
The markup currently has some errors and warning that needs getting sorted out.
- Kindly let us see to it:
errors

## [OPTIONAL] - REDUCE - The use of comments in your code
- Normally comments are a good thing, they help us remember bits of our code when we come back to it. However, I'd recommend we minimise overusing comments as it makes the codebase ugly. 
It’s better to delete the majority of your comments. If you need to review a previous version of your code, git history is perfect for this, thank you,

## 
Please make these changes and submit another review request. Thank you!
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by ElijahTobs** [comment]
I have implemented all required and optional changes

**Comment by sumancrest0001** [comment]
STATUS: CHANGES REQUIRED
Great job and those improvements already made your project more beautiful. :clap: :clap: 
But still, we need to make improvements to get :+1:  for the project.

Kindly make the following changes.

[REQUIRED] Instead of the account link we have to add **log in button** and nav bar must be sticky at the top. 
**Please have a look :point_down: :point_down:** 

Screenshot from 2020-04-05 19-56-18

**In original project :point_down:** 
Screenshot from 2020-04-05 19-56-58

File: index.html [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+    
+    <nav class="nav-bar">
+        <span class="right-nav-bar">
+          <i class="fas fa-bars bars"></i>
+          <i class="fas fa-search search"></i>
+          <span>SPACE & COSMOS</span>
+        </span>
+        <span class="center-nav-bar">
+          <img src="./img/nytLogo.PNG" alt="nytLogo">
+        </span>
```
Comment (sumancrest0001) [inline index.html]: [REQUIRED] The logo image should be a link. Not just only an image.

**Comment by sumancrest0001** [comment]
[REQUIRED] We have to add a sticky footer section at the bottom.

In original project :point_down:  :point_down: 
Screenshot from 2020-04-05 20-01-25

[link to original page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)

**Comment by sumancrest0001** [comment]
**Please make above changes and submit for another review.**

Need help? message me on slack **Suman Shrestha**

**Comment by uryelah** [comment]
Hello **Samuel Ebeagu** & **Elijah Ayandokun**, I'm Sarah, your last reviewer for this PR 👩‍🚀  

**Good job with the fixes from the last reviews! 🌟**

The project is looking great, I love how you also added the hover interactions and paid attention to font size and style.

------
## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

----

### Optional Changes

- [ ] The navbar and fixed footer should both take 100% of the screen width. Sometimes you can miss that according to your screen size. A fast trick to see how your page looks in a slightly wider screen is clicking `ctrl+`, or you could use your browser responsiveness tools.

Notice how there's space in the sides and you can even see the box-shadow 'leaking':
Screenshot_2020-04-06 NYTClone
Screenshot_2020-04-06 NYTClone(2)

- [ ] Make the icons with the circular border bellow the article all have a circle around then by setting a fixed width and height. Notice how the Facebook icon bellow has an ellipse instead:
Screenshot_2020-04-06 NYTClone(1)
----

**Keep being awesome devs and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**